django/core/management/commands/makemigrations.py

```diff
@@ -223,7 +223,7 @@ def write_migration_files(self, changes):
                         directory_created[app_label] = True
                     migration_string = writer.as_string()
                     with open(writer.path, "wb") as fh:
-                        fh.write(migration_string)
+                        fh.write(migration_string.encode('utf-8'))
```

Comment:
Can we simply use `open()` w/ `w` instead of `wb` and not have `encode()` here (same for other locations)?